migrations/v2.js

```diff
@@ -266,14 +266,14 @@ describe('core - update to v2.1.0', async () => {
 
   mutateContent('core - update _accessibility._isEnabledOnTouchDevices', async (content) => {
     const originalValue = _.get(config, '_accessibility._isEnabledOnTouchDevices');
-    _.set(config, '_accessibility.accessibilityToggleTextOn', originalValue || 'Turn accessibility on?');
+    _.set(config, '_accessibility._isDisabledOnTouchDevices', originalValue || 'Turn accessibility on?');
```

Comment:
`isEnabled` suggests a boolean rather than an instruction text.

Reply:
Coffee was clearly needed, apologies managed to confuse myself. That has been amended to appropriate values.